### ISSUE TITLE
Fix ShareExtensionHelper warnings

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#17188]
```
⚠️  /Users/vagrant/git/Client/Frontend/Share/ShareExtensionHelper.swift:146:33: 'kUTTypeFileURL' was deprecated in iOS 15.0: Use UTTypeFileURL or UTType.fileURL (swift) instead.
            return url.isFile ? kUTTypeFileURL as String : kUTTypeURL as String
                                ^
⚠️  /Users/vagrant/git/Client/Frontend/Share/ShareExtensionHelper.swift:146:60: 'kUTTypeURL' was deprecated in iOS 15.0: Use UTTypeURL or UTType.url (swift) instead.
            return url.isFile ? kUTTypeFileURL as String : kUTTypeURL as String
                                                           ^
⚠️  /Users/vagrant/git/Client/Frontend/Share/ShareExtensionHelper.swift:149:62: 'kUTTypeURL' was deprecated in iOS 15.0: Use UTTypeURL or UTType.url (swift) instead.
        return activityType == nil ? browserFillIdentifier : kUTTypeURL as String
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-7707)

**Comment by data-sync-user** [comment]
➤ Laurie Marceau commented:

For QA it can be tested that the share view, accessible through the share button works as expected

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Verified as fixed on v9000 (36168), with iPhone SE (16.6.1).
